definitions/procedures/foreman_tasks/fetch_tasks_status.rb

```diff
@@ -0,0 +1,15 @@
+module Procedures::ForemanTasks
+  class FetchTasksStatus < ForemanMaintain::Procedure
+    metadata do
+      for_feature :foreman_tasks
+      description 'Fetch tasks status and wait till they finish'
+      param :state, :required => true
+    end
+
+    def run
+      with_spinner("wait till #{@state} tasks get finish") do |spinner|
```

Comment:
`till` is quite informal, let's use `until` instead. I would also recommend using `ing` from, as we are describing, what's happening at the moment, and instead of `get finish`, it would sound better to me to `waiting for 2 tasks to finish` or `waiting until tasks get finished`.

I would recommend `waiting for #{@state} tasks to finish`